src/go/rdctl/cmd/root.go

```diff
@@ -19,35 +19,22 @@ package cmd
 
 import (
 	"bytes"
-	"encoding/json"
 	"fmt"
 	"io/ioutil"
-	"log"
 	"net/http"
 	"os"
-	"path/filepath"
-	"strconv"
 	"strings"
 
 	"github.com/spf13/cobra"
+
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/config"
 )
 
 type APIError struct {
 	Message          *string `json:"message,omitifempty"`
 	DocumentationUrl *string `json:"documentation_url,omitifempty"`
```

Comment:
Just realized this is a typo — it should be `omitempty` not `omitifempty`. Probably separate PR though.

Reply:
best to get it fixed now